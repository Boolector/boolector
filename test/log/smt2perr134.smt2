(assert (and (_ zero_extend
