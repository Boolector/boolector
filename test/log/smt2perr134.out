log/smt2perr134.smt2:1:10: expected '(' at 'and' before '(_ zero_extend'
